baybe/searchspace/core.py

```diff
@@ -66,6 +66,13 @@ class SearchSpace(SerialMixin):
     continuous: SubspaceContinuous = field(factory=SubspaceContinuous.empty)
     """The (potentially empty) continuous subspace of the overall search space."""
 
+    def __str__(self) -> str:
+        start_bold = "\033[1m"
+        end_bold = "\033[0m"
+        print(f"\n\n{start_bold}***** SEARCH SPACE *****\n {end_bold}")
+
+        return str(self.discrete) + str(self.continuous)
```

Comment:
the search space type `via self.type` comes to mind as an additonal interesting info that could be printed even before the other two